package/src/bin/omicron-package.rs

```diff
@@ -188,6 +192,8 @@ async fn do_for_all_rust_packages(
         // If this is a Rust package...
         if let PackageSource::Local { rust: Some(rust_pkg), .. } = &pkg.source {
             let plan = if rust_pkg.release { &mut release } else { &mut debug };
+            // Add the package name to the plan
```

Comment:
Could you add a comment here about why this is necessary? Just a link to the issue or this PR would do.

Also worth noting somewhere (if not already done) that when `source.rust` is defined, the table key (e.g. the `omicron-nexus` in `[package.omicron-nexus]`) is the name of the corresponding Rust crate.

Reply:
It may also be useful to link to https://github.com/rust-lang/cargo/issues/8157 ?

Reply:
> Also worth noting somewhere (if not already done) that when source.rust is defined, the table key (e.g. the omicron-nexus in [package.omicron-nexus]) is the name of the corresponding Rust crate.

This is mentioned here:
https://github.com/oxidecomputer/omicron/blob/d01b2ee56a24e3ec36a75c91d7e4959c9bc87c16/package-manifest.toml#L16-L17

but I agree it's non-obvious and it wouldn't hurt to mention it in other places where a reader might not know.

Reply:
To be clear, in the comment here. I know the PR has it!

Reply:
https://github.com/oxidecomputer/omicron/pull/7218/commits/acbe39a60eea69d20926a68b480fb04b4df03bd3 does most of this; I think the particular detail about how to write package-manifest.toml is documented in the file itself:

https://github.com/oxidecomputer/omicron/blob/d01b2ee56a24e3ec36a75c91d7e4959c9bc87c16/package-manifest.toml#L12-L17

But I think I improved the comments to make it clearer in that code's context.